go.mod

```diff
@@ -1,144 +1,153 @@
 module github.com/athanorlabs/go-relayer
 
-go 1.19
+go 1.20
 
 require (
 	github.com/athanorlabs/go-p2p-net v0.0.0-20230126054330-40336771ef86
```

Comment:
This repo is bit-rotting faster than we can maintain it. We'd have to make major updates to code we don't use anywhere to update this dependency.